### PR TITLE
Check if user already exists before creating user

### PR DESCRIPTION
**Pull request by Janfred**
This PR is implementing a solution for #27

It checks if a user already exists before executing the `create_user` script.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -27,7 +27,12 @@ runcommand() {
 }
 
 # create passwd entry for current uid, fix HOME variable
-create_user $(id -u)
+# only execute, if the current uid does not exist.
+if id $(id -u) >/dev/null 2>/dev/null; then
+    echo Not creating user with uid $(id -u) because he already exists
```
Comment (gebart) [inline run.sh]: I don't want to be picky, but could you rewrite this message in a gender neutral form, please?

Comment (gebart) [inline run.sh]: Also, please add double quotation marks around the message.

**Comment by kaspar030** [comment]
- do we need the error message at all? (just asking)
- could you do "id -u" once at the beginning and put it in a variable? No need to call id more than once.

**Comment by Janfred** [comment]
@kaspar030 The message was introduced by me for debugging, I thought it might be useful for production, but I can remove it.

The reason why I haven't put `id -u` in a variable is, that I test if `id -u` exits with an error return code. I could use $? but this is imho not good coding style and leads to difficult debugging process once another call is added before the `if`.

**Comment by smlng** [comment]
please squash, otherwise I think its a good addition, and I ran into such an issue already. @kaspar030 you think its fine, too?

**Comment by smlng** [comment]
oh wait, we can you GitHub merge/squash here

**Comment by kaspar030** [comment]
> you think its fine, too?

Yes!

**Pull request merged**